.travis.yml

```diff
@@ -33,19 +35,28 @@ cache:
 addons:
   postgresql: "9.3"
 
+before_install:
+  - |
+    if [ $TRAVIS_PHP_VERSION = '5.4' ]; then
```

Comment:
This needs to be added:

```yaml
before_install:
  - composer config -g github-oauth.github.com "$GITHUB_TOKEN"
```

See https://github.com/yiisoft/yii2/issues/12987#issuecomment-260254948.